carla-carma-integration/carla_carma_bridge/src/carla_carma_bridge/carma_carla_guidance

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# Copyright (C) 2021 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+import rospy
+from cav_msgs.msg import RouteEvent
+from cav_msgs.msg import Plugin
+
+route_selected = False
+plugins_activated = False
+
+def check_route_status(route_event_msg):
+    if route_event_msg.event == 1:
+        print("Route Selected")
+        route_selected = True
```

Comment:
The `route_selected` should be set as global parameter before assign value since the `check_route_status` is a callback function.

Reply:
I think this comment is handled, right? Looks like the global call on line 25 should do what you've described.